compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/TileAttention.cpp

```diff
@@ -158,6 +159,17 @@ struct TileAttentionPass : public TileAttentionBase<TileAttentionPass> {
   void runOnOperation() override;
 };
 
+struct ConvertAttentionToOnlineAttentionPass
+    : public ConvertAttentionToOnlineAttentionBase<
+          ConvertAttentionToOnlineAttentionPass> {
+  void getDependentDialects(DialectRegistry &registry) const override {
+    registry
+        .insert<affine::AffineDialect, IREE::LinalgExt::IREELinalgExtDialect,
+                linalg::LinalgDialect, tensor::TensorDialect>();
```

Comment:
Move this to TD?

Reply:
I think LinalgExt is still using the old pass td style. Can that support this?

Reply:
I think so

Reply:
I tried to do it, but it seems to cause a lot of issues. I'm going to keep it like this for now, since other passes in LinalgExt do this. I'll send a patch to modernize the passes as a followup.